### PR TITLE
PR guidelines

### PR DESCRIPTION
**Pull request by nb1701**

### CODE REVIEW

File: docs/contributor-guide/developer-guide/technical-contribution-guide.md [hunk 1]
```diff
@@ -147,25 +147,30 @@ If you're editing files that others have merged changes to you may need to resol
...
-
-If the pull request does not require additional changes, the reviewer should merge it immediately after approving. Otherwise, once they have addressed all comments marked `(blocking)` or `nit`, the pull request author should either merge if able or re-request review and merging from a maintainer if not. Authors are encouraged to at least reply to `(non-blocking)` and `(if-minor)` comments if they do not address them with code changes.
+You can select the reviewer you feel most has context on your PR. If you want a round robin review, add `civiform/developers` as a reviewer and a team member will be chosen. More details on [how to add roundrobin reviews here](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team). Once that is done you can add the name of the team (`civiform/developers`) in the reviewer box and it will auto assign someone. Alternatively, you may use one of the subsystem groups to get a domain expert to review. These are `civiform/auth-system`, `civiform/core-application`, `civiform/deployment-system`, `civiform/developer-tools`, and `civiform/export-api`. There may be others as time goes on.
```
Comment (avaleske) [inline docs/contributor-guide/developer-guide/technical-contribution-guide.md:173]: very nit, but can we format the domain experts as a list so it's easier to scan?

And should we link to where the domain experts are defined so people know how to add themselves if they want?

File: docs/contributor-guide/developer-guide/technical-contribution-guide.md [hunk 2]
```diff
@@ -147,25 +147,30 @@ If you're editing files that others have merged changes to you may need to resol
...
+- Remove checkboxes from the PR template that don't apply. Ensure all checkboxes are checked before a PR is merged.
+- If a PR is inconsequential for deployments, like changes to github templates or actions, code changes without user-visible effects, etc., add the `ignore-for-release` label, so it is not included in the release notes. If they are deployment tool-specific or infrastructure-specific (read: dev tooling) changes, add `deploy` or `infrastructure` labels, as these will also get removed from the release notes.
+- If your PR includes a database evolution, include the `database` label.
+- A PR should be opened as Draft until it is ready for review.
+- A PR is ready for review when it is code complete, unit and browser tests are created (where appropriate), `bin/fmt` has been run, all checkboxes listed in the PR template that are relevant are checked and completed, all PR checks are passing, and manual testing has been performed.
+- Manual testing should ideally be performed with both Chrome and Firefox if your PR includes a front-end change.
```
Comment (rockycodes) [inline docs/contributor-guide/developer-guide/technical-contribution-guide.md:157]: suggestion (non-blocking): Some accessibility functionality doesn't work on Firefox on a mac (eg. tabbing through buttons). We can still recommend testing on Firefox, but might be good to make a note of this.

File: docs/contributor-guide/developer-guide/technical-contribution-guide.md [hunk 3]
```diff
@@ -147,25 +147,30 @@ If you're editing files that others have merged changes to you may need to resol
...
+- If appropriate, add instructions in PR comments for reviewers on how to manually test it.
+- Press the "re-request review" button (the "arrows pointing in a circle" icon next to the reviewer's name in the Reviewers section on the Conversation tab) when you've fixed all the things mentioned in the comments and are ready for the reviewers to re-review.
```
Comment (rockycodes) [inline docs/contributor-guide/developer-guide/technical-contribution-guide.md:159]: suggestion (non-blocking): Add language about posting in the the #pull-request-discuss channel and pinging the reviewer (optional?)

File: docs/contributor-guide/developer-guide/technical-contribution-guide.md [hunk 4]
```diff
@@ -147,25 +147,30 @@ If you're editing files that others have merged changes to you may need to resol
...
+- Resolve any minor, non-blocking comments. Blocking comments should only be resolved by the reviewer.
+
+#### Review guidelines
+- If you are tagged as a reviewer, aim to have a turnaround time of no more than 24 hours. Responding to any subsequent chagnes to your comments should be a priority and addressed ASAP.
```
Comment (rockycodes) [inline docs/contributor-guide/developer-guide/technical-contribution-guide.md:163]:
```suggestion
- If you are tagged as a reviewer, aim to have a turnaround time of no more than 24 hours. Responding to any subsequent changes to your comments should be a priority and addressed ASAP.
```

**Review by rockycodes — APPROVED** [review]
great write-up!

**Pull request merged**